src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -219,7 +251,7 @@ public AcquireTokenSilentParameterBuilder AcquireTokenSilent(IEnumerable<string>
         /// specify extra query parameters
         ///
         /// </remarks>
-        public AcquireTokenSilentParameterBuilder AcquireTokenSilent(IEnumerable<string> scopes, string loginHint)
+        public AcquireTokenSilentParameterBuilder AcquireTokenSilentWithLoginHint(IEnumerable<string> scopes, string loginHint)
```

Comment:
>AcquireTokenSilentWithLoginHint [](start = 50, length = 31)

Same comment as above. perhaps we should add deprecation messages and not completely remove? Though given we move to v4 perhaps ok. Lots of samples needs to be updated based on these changes.

Reply:
We cannot release a new version 3 weeks after the GA and require everybody to update absolutely all their code.